app/src/main/java/com/tehmou/rxbookapp/fragments/RepositoriesFragment.java

```diff
@@ -48,20 +47,22 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container,
     @Override
     public void onViewCreated(View view, Bundle savedInstanceState) {
         super.onViewCreated(view, savedInstanceState);
-        repositoriesView = (RepositoriesView) view.findViewById(R.id.repositories_view);
+        repositoriesViewBinder = new RepositoriesView.ViewBinder(
+                (RepositoriesView) view.findViewById(R.id.repositories_view),
```

Comment:
Added a test for the null within the constructor. I guess checking for null here is not necessary since it would only cause the exception earlier